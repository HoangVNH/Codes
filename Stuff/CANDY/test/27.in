10
36
VVVVVVVDDDDDVVVDDDDDDXXXXXXXXXXXXXXV
50
DDDDVDDDDXXXDDDDDDDDDXXXXXDDDDDDDDDDXVDDDDDDXXXXXX
40
VVVVVDDDDDDXXXXXXXXXXXXXXXVVVVXDDDDDDDVV
38
XXXXXXXXDDDDDVVVVVVVVVVXXXXXXXXXVVVVXX
20
DVVVVXXXXDDDXXDDDXXD
27
VVVVVVVVVDDDDXXXXXVVVVVVXXD
37
DDDDDDDDDDDDDDDXXXXXXXXDDDDDDDDDDVVVV
47
XDXXXXDDDDDDDVVVDDDDDDDDDDDDDDDDXXXXXDVVVVVVVVV
49
XXXXVVVDDDDDDDDDDDDVVVVVVVDDDDDXXXXXDDDDDDDDDDDDD
41
VVVVVVVVXXXXXVVVVVVVVVVVVVVVDDDDDDDDDVVVV
