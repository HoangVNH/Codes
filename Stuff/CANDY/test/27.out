35
49
39
37
18
25
36
46
48
39
